openwisp_radius/tasks.py

```diff
@@ -99,5 +101,9 @@ def send_login_email(accounting_data):
             'call_to_action_url': one_time_login_url,
             'call_to_action_text': _('Manage Session'),
         }
+        if hasattr(settings, 'SESAME_MAX_AGE'):
+            context.update(
+                {'sesame_max_age': str(timedelta(seconds=settings.SESAME_MAX_AGE))}
```

Comment:
Instead of passing in this format you can use [`timesince`](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#timesince) django template tag which is more readable than this.

Reply:
Yes but that filter works with date instance, Here we only getting `SESAME_MAX_VALUE` as integer 🤔

Reply:
> Yes but that filter works with date instance, Here we only getting `SESAME_MAX_VALUE` as integer thinking

This works :point_down: 

```python
# SESAME_MAX_AGE =  60 * 60
# SESAME_MAX_AGE =  60 * 60 * 60
sesame_max_age = datetime.datetime.utcnow() - datetime.timedelta(seconds=settings.SESAME_MAX_AGE)
{{sesame_max_age | timesince}} # In templates
```
![Screenshot from 2022-02-09 12-44-59](https://user-images.githubusercontent.com/56113566/153140752-97e28fd1-9731-46e9-a868-a3a96a276d85.png)

![Screenshot from 2022-02-09 12-45-17](https://user-images.githubusercontent.com/56113566/153140711-8003de1a-c583-4924-b479-4150c90806c0.png)

Reply:
Another solution is `naturaltime` but it depends on `django.contrib.humanize`